src/core/map/tile.rs

```diff
@@ -237,6 +233,15 @@ mod tile_tests {
 		}
 	}
 
+	impl std::clone::Clone for TileType<DummyIO> {
```

Comment:
TileType's shouldn't really be cloneable, if its wanted for tests then it would be best to put it behind a test configuration so it doesn't get compiled in when its not a test.  :-)

Reply:
Oh man, it's been a while since I looked at any of this. Anyways, it does look to me like the clone implementation is inside of the test config, and none of it is marked pub. Is there some other thing I have to do to make it only visible to the tests? Looking at the test that uses it though it'd probably be possible to remove the clone requirement anyways.

Reply:
This will all be getting migrated to https://github.com/OvermindDL1/over_civ/ as well as a more permanent home, this project was a sandbox/testing project after all, so it seems logical to pull it out into its own project.  :-)